src/stores/file_store.rs

```diff
@@ -8,10 +8,12 @@ pub async fn store_messages(messages: Vec<LogMessage>, pool: &Pool) {
         .await
         .unwrap()
         .interact(move |connection| {
-            diesel::insert_into(schema::loggaroo::log_entry::table)
-                .values(messages)
-                .execute(connection)
-                .unwrap();
+            for messages in messages.chunks(5000) {
+                diesel::insert_into(schema::loggaroo::log_entry::table)
+                    .values(messages)
+                    .execute(connection)
+                    .unwrap();
```

Comment:
Add await

Reply:
what exactly should I await?

there is no async function. If there was the code wouldn't work since futures are lazy

Reply:
Ok but didnt you say something about awaiting in the school?

Reply:
i was talking about this await

```rust
    pool.get()
        .await
        .unwrap()
        .interact(|connection| {
            diesel::insert_into(schema::loggaroo::file::table)
                .values(files)
                .execute(connection)
                .unwrap();
        })
        .await // <- this
        .unwrap();
```

i thought we might be able to run the `.execute()` in parallel since i thought we would have an await after each `.execute()`
but the await is already coming after the `.executes()`.

maybe it still would make sence to use multiple connections since the `.execute()` is probably blocking